src/services/leafletWatchHelpers.js

```diff
@@ -20,7 +20,17 @@ angular.module("leaflet-directive")
   to set watching to once and its watch depth.
   */
   var _maybeWatch = function(scope, thingToWatchStr, watchOptions, initCb){
-      return _maybe(scope, '$watch', thingToWatchStr, watchOptions, initCb);
+      var localWatchOptions, watchMethod;
```

Comment:
By heavily I mean I use this to disable watching and is in conjuction with [directiveControls](https://github.com/tombatossals/angular-leaflet-directive/blob/master/src/services/leafletData.js#L23) and [here](https://github.com/tombatossals/angular-leaflet-directive/blob/8f0cb1bd9dffbf0b272544b699db54893ec40f3b/examples/js/controllers/MixedMOverlaysMarkersNestedNoWatchController.js#L9-L20)